packages/mdc-ripple/_mixins.scss

```diff
@@ -145,8 +145,7 @@
 // Simple mixin for activated states which automatically selects opacity values based on whether the ink color is
 // light or dark.
 @mixin mdc-states-activated($color, $has-nested-focusable-element: false) {
-  $opacity-map: mdc-states-opacities_($color);
-  $activated-opacity: map-get($opacity-map, "activated");
+  $activated-opacity: mdc-states-opacity($color, activated);
```

Comment:
I wanted to stay consistent with the usage pattern we use for `mdc-theme-prop`, where `primary`, `secondary`, etc. aren't surrounded in quotes when used as values.

Either should technically work, though unquoted values could be seen as a bit sloppy since they may be coerced to color if recognized as one (e.g. `black`), but even this is harmless as long as the usage is consistent both in the map's keys and in `map-get` invocations.

If quoting is something we want to eventually encourage instead, we'd want to do so for `mdc-theme-prop` as well though.

Reply:
Ok leaving it without quotes LGTM